test/Microsoft.AspNetCore.Mvc.Core.Test/PhysicalFileResultTest.cs

```diff
@@ -236,6 +421,35 @@ protected override Stream GetFileStream(string path)
                     return new MemoryStream(Encoding.UTF8.GetBytes("FilePathResultTestFile contents�"));
                 }
             }
+
+            protected override FileInfo GetFileInfo(string path)
+            {
+                var lastModified = DateTimeOffset.UtcNow;
+                return new FileInfo
+                {
+                    Exists = true,
+                    Length = 34,
+                    LastModified = new DateTimeOffset(lastModified.Year, lastModified.Month, lastModified.Day, lastModified.Hour, lastModified.Minute, lastModified.Second, TimeSpan.FromSeconds(0))
+                };
+            }
+        }
+
+        private class TestSendFileFeature : IHttpSendFileFeature
+        {
+            public string name = null;
```

Comment:
Got this from [here](https://github.com/aspnet/HttpAbstractions/blob/dev/test/Microsoft.AspNetCore.Http.Extensions.Tests/SendFileResponseExtensionsTests.cs#L36) Should I change that as well?